ublox_derive/src/output.rs

```diff
@@ -512,6 +571,20 @@ pub fn generate_code_to_extend_bitflags(bitflags: BitFlagsMacro) -> syn::Result<
         },
     };
 
+    let serialize_fn = format_ident!("serialize_{}", repr_ty.to_token_stream().to_string());
+    let serde = quote! {
+        impl serde::Serialize for #name {
+            fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+            where
+                S: serde::Serializer,
+            {
+                // serializing debug gives naming
+                // serializer.serialize_str(format!("{:?}", self).as_str())
```

Comment:
```suggestion
```

Reply:
could we keep these, was thinkin of having them in CLI as an option to toggle